readme.md

```diff
@@ -69,6 +69,7 @@ Click the type names for complete docs.
 - [`Merge`](source/merge.d.ts) - Merge two types into a new type. Keys of the second type overrides keys of the first type.
 - [`MergeExclusive`](source/merge-exclusive.d.ts) - Create a type that has mutually exclusive properties.
 - [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given properties.
+- [`PartialDeep`](source/partial-deep.d.ts) - Create a type from another type with all properties set to optional and the nested properties of all properties set to optional, as opposed to the [`Partial<T>`](https://github.com/Microsoft/TypeScript/blob/2961bc3fc0ea1117d4e53bc8e97fa76119bc33e3/src/lib/es5.d.ts#L1401-L1406) type which constructs a type with properties of T that have a depth of 1 set to optional.
```

Comment:
Would be good to write this more succinctly.

Reply:
Fixed in c48b576